docs/mkdocs/api/authentication.md

```
@@ -1,0 +1,162 @@
+  },
+  "message": "User registered. Please verify your email."
+}
+```
+
+**Next**: Check the MailHog UI (`http://localhost:8025`) for a verification email.
+
+## Step 2: Verify Email
+
+After registration, a verification email is sent to the provided email address. In the development environment, access the email via MailHog.
+
+1.  Navigate to `http://localhost:8025`.
```

markdownlint-cli2 (0.17.2)

Spaces after list markers
Expected: 1; Actual: 2

(MD030, list-marker-space)